### PR TITLE
#520 Qulice does not enforce Checkstyle rule NoLineWrap

### PR DESCRIPTION
**Pull request by HDouss**
#520 Qulice does not enforce Checkstyle rule NoLineWrap
Added check rule in checks.xml
Added a source file to test and a test method in CheckstyleValidatorTest

### CODE REVIEW

**Comment by HDouss** [comment]
@davvd Please assign a CR

**Comment by davvd** [comment]
@hdouss Thanks for the pull request, let me find a reviewer..

**Comment by davvd** [comment]
@dmzaytsev it's yours,please review

**Comment by dmzaytsev** [comment]
@HDouss looks good

**Comment by dmzaytsev** [comment]
@rultor merge

File: qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java [hunk 1]
```diff
@@ -207,6 +207,19 @@ public void reportsErrorWhenParameterObjectIsNotDocumented()
     }
 
     /**
+     * Real.
```
Comment (krzyk) [inline qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java]: @HDouss you forgot to write a Javadoc :-)

File: qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java [hunk 2]
```diff
@@ -207,6 +207,19 @@ public void reportsErrorWhenParameterObjectIsNotDocumented()
...
+     * @throws Exception xhen error.
```
Comment (krzyk) [inline qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java]: @HDouss typo: xhen

File: qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml [hunk 3]
```diff
@@ -257,6 +257,11 @@
         <module name="UnusedImports"/>
         <module name="ImportOrder"/>
 
+        <!--
+        Checks that imports and package are not line wrapped
```
Comment (krzyk) [inline qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml]: @HDouss this comment is not needed

**Comment by krzyk** [comment]
@HDouss few comments from me

**Comment by HDouss** [comment]
@krzyk Absolutely right :) That is done !

**Comment by HDouss** [comment]
@krzyk Please initiate merge.

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by davvd** [comment]
@dmzaytsev Thanks a lot, I just topped your account for **17 mins**, transaction ID `72541213`, total time was 3 hours and 44 mins; extra minutes for review comments (c=2); +17 added to your rating, at the moment it is: [+975](http://www.netbout.com/b/35166?open=rating)

**Comment by davvd** [comment]
@rultor deploy now pls

**Pull request merged**